From 58646f62bc37cadbf226c6f6b32c9185097b9098 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 25 Jul 2017 11:44:35 +0800
Subject: [PATCH 2210/5242] MLK-16075-15 gpu: imx: dpu: fetchdecode: Specify
 DPU_VPROC_CAP_FETCHECO0/1 cap

commit  378082ed8535f1cfbe7d6ec9029a5eb23e4b7e2a from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch specifies DPU_VPROC_CAP_FETCHECO0/1 video processing
capabilities of fetchdecode.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-fetchdecode.c |   18 ++++++++++++------
 1 file changed, 12 insertions(+), 6 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-fetchdecode.c b/drivers/gpu/imx/dpu/dpu-fetchdecode.c
index 20391b1..df1f454 100644
--- a/drivers/gpu/imx/dpu/dpu-fetchdecode.c
+++ b/drivers/gpu/imx/dpu/dpu-fetchdecode.c
@@ -25,15 +25,21 @@
 #define FD_NUM_V2			2
 
 static const u32 fd_vproc_cap_v1[FD_NUM_V1] = {
-	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4,
-	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5,
-	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4,
-	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5,
+	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4 |
+	DPU_VPROC_CAP_FETCHECO0,
+	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5 |
+	DPU_VPROC_CAP_FETCHECO1,
+	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4 |
+	DPU_VPROC_CAP_FETCHECO0,
+	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5 |
+	DPU_VPROC_CAP_FETCHECO1,
 };
 
 static const u32 fd_vproc_cap_v2[FD_NUM_V2] = {
-	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4,
-	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5,
+	DPU_VPROC_CAP_HSCALER4 | DPU_VPROC_CAP_VSCALER4 |
+	DPU_VPROC_CAP_FETCHECO0,
+	DPU_VPROC_CAP_HSCALER5 | DPU_VPROC_CAP_VSCALER5 |
+	DPU_VPROC_CAP_FETCHECO1,
 };
 
 #define PIXENGCFG_DYNAMIC		0x8
-- 
1.7.9.5

